deployment/helm/templates/webhook-configuration.yaml

```diff
@@ -0,0 +1,137 @@
+{{- if .Values.rbac.enabled }}
```

Comment:
`.Values.webhookConfiguration.enabled`

Reply:
IIUC, the `serviceAccount` is only injected into the template if `rbac.enabled` (sic.)

However, I'm unsure whether there's any benefit to RBAC for the Webhook.... The Deployment (Pods) need no permissions

Reply:
it looks like you and @kate-goldenring are discussing this on another thread, so i can resolve this ... but it still seems to me like this file should be wrapped in a `{{- if .Values.webhookConfiguration.enabled }}`

Reply:
You're, of course, correct (and it now is). When I first read your comment and didn't read the line number. There's a later occurence of `{{- if .Values.rbac.enabled }}` for the service account and I thought you were referring to that. Apologies!

Reply:
no worries ... teamwork makes the dreamwork :)

Reply:
Searches for the groan emoticon :smile: